ord_schema/units.py

```diff
@@ -237,3 +237,38 @@ def format_message(message: ord_schema.UnitMessage) -> Optional[str]:
         txt += f'(± {message.precision:.7g}) '
     txt += _UNIT_SYNONYMS[type(message)][message.units][0]
     return txt
+
+
+def compute_solute_quantity(
```

Comment:
this might require some updates after #610 goes in

Reply:
This was okay but the brine() macro needed updates to use the new SATURATED concept.